setup.py

```diff
@@ -52,15 +52,16 @@
     long_description_content_type="text/markdown",
     long_description=long_description,
     name=PROJECT_SLUG,
-    packages=[SOURCE_DIR],
+    package_dir={"": "src"},
+    packages=find_packages(where="src"),
     python_requires=">=3.6,<4",
     url=f"https://github.com/ARMmbed/{PROJECT_SLUG}",
     version=__version__,
     entry_points={
         "console_scripts": [
-            f"mbedtools={SOURCE_DIR}.cli:cli",
-            f"mbed-tools={SOURCE_DIR}.cli:cli",
-            f"mbed_tools={SOURCE_DIR}.cli:cli",
+            "mbedtools=mbed_tools.cli.cli:cli",
```

Comment:
.cli.cli:cli looks funny. Is that correct? so many .cli?

Reply:
I change it later when moving the cli's to top level.